BookReader/BookReader.js

```diff
@@ -1712,7 +1719,7 @@ BookReader.prototype.enterFullscreen = function() {
     this.refs.$brContainer.animate({opacity: 1}, 400, 'linear');
 
     this._fullscreenCloseHandler = function (e) {
-        if (e.keyCode === 27) this.exitFullScreen();
+        if (e.keyCode === 27) this.toggleFullscreen();
```

Comment:
Suggestion out of scope of this PR, but suggestion, possibly for `Hacktoberfest`, replacing magic number 27 w/ a const
```suggestion
        const escKey = 27;
        if (e.keyCode === escKey) this.toggleFullscreen();
```